crates/edr_op/src/chains.rs

```diff
@@ -0,0 +1,80 @@
+use std::sync::LazyLock;
+
+use edr_evm::hardfork::{Activations, ChainConfig, ForkCondition};
+use op_revm::OpSpecId;
+
+/// OP Mainnet chain ID
+pub const OP_MAINNET_CHAIN_ID: u64 = 0xa;
+
+/// OP Mainnet chain config
+///
+/// https://github.com/ethereum-optimism/superchain-registry/blob/51804a33655ddb4feeb0ad88960d9a81acdf6e62/superchain/configs/mainnet/op.toml
+pub static OP_MAINNET_CONFIG: LazyLock<ChainConfig<OpSpecId>> = LazyLock::new(|| ChainConfig {
+    name: "op-mainnet".into(),
```

Comment:
Is it okay to change the name of the existing network? (`mainnet` -> `op-mainnet`)

Reply:
Yup. It's in fact more accurate! https://chainlist.org/chain/10